.github/ISSUE_TEMPLATE.md

```diff
@@ -9,6 +9,8 @@ https://petitions.classicpress.net/
 
 For more details about our governance and change planning process, see:
 https://www.classicpress.net/democracy/
+
+Please find guidelines on how to contribute to this project in our [CONTRIBUTING.md](../CONTRIBUTING.md)
```

Comment:
I'd like to take this further. This should be a checkbox at the top of the issue template, and we should take a look at what can be removed from the template once that's added.